src/components/error-dialog/container.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
```

Comment:
Why do we need a container if it's not providing any functionality?

Reply:
I was thinking the same, but wondered if at any point in the future we would be pulling in state or actions if this dialog was to be used in other scenarios. Maybe thinking a little too far ahead for that. I can do a follow up to this PR and remove the container